addons/hr/models/res_users.py

```diff
@@ -1,21 +1,156 @@
+    marital = fields.Selection(related='employee_id.marital', readonly=False, related_sudo=False)
+    spouse_complete_name = fields.Char(related='employee_id.spouse_complete_name', readonly=False, related_sudo=False)
+    spouse_birthdate = fields.Date(related='employee_id.spouse_birthdate', readonly=False, related_sudo=False)
+    children = fields.Integer(related='employee_id.children', readonly=False, related_sudo=False)
+    emergency_contact = fields.Char(related='employee_id.emergency_contact', readonly=False, related_sudo=False)
+    emergency_phone = fields.Char(related='employee_id.emergency_phone', readonly=False, related_sudo=False)
+    visa_no = fields.Char(related='employee_id.visa_no', readonly=False, related_sudo=False)
+    permit_no = fields.Char(related='employee_id.permit_no', readonly=False, related_sudo=False)
+    visa_expire = fields.Date(related='employee_id.visa_expire', readonly=False, related_sudo=False)
+    google_drive_link = fields.Char(related='employee_id.google_drive_link', readonly=False, related_sudo=False)
+    additional_note = fields.Text(related='employee_id.additional_note', readonly=False, related_sudo=False)
+
+
+    def __init__(self, pool, cr):
+        """ Override of __init__ to add access rights.
+            Access rights are disabled by default, but allowed
+            on some specific fields defined in self.SELF_{READ/WRITE}ABLE_FIELDS.
+        """
+        hr_readable_fields = [
+            'employee_ids',
+            'employee_id',
+            'active',
+            'parent_id',
+            'child_ids',
+        ]
+
+        hr_writable_fields = [
+            'display_name',
+            'job_title',
+            'work_phone',
+            'phone',
+            'mobile_phone',
+            'work_email',
+            'category_ids',
+            'department_id',
+            'address_id',
+            'work_location',
+            'parent_id',
+            'coach_id',
+            'notes',
+            'address_home_id',
+            'is_address_home_a_company',
+            'km_home_work',
+            'employee_bank_account_id',
+            'employee_country_id',
+            'identification_id',
+            'passport_id',
+            'gender',
+            'birthday',
+            'place_of_birth',
+            'country_of_birth',
+            'marital',
+            'spouse_complete_name',
+            'spouse_birthdate',
+            'children',
+            'emergency_contact',
+            'emergency_phone',
+            'visa_no',
+            'permit_no',
+            'visa_expire',
+            'google_drive_link',
+            'additional_note',
+        ]
+
+        init_res = super(User, self).__init__(pool, cr)
+        # duplicate list to avoid modifying the original reference
+        type(self).SELF_READABLE_FIELDS = type(self).SELF_READABLE_FIELDS + hr_readable_fields + hr_writable_fields
```

Comment:
Maybe put in a method 
``` python
def _extend_readable_fields(self, fields)
  type(self).SELF_READABLE_FIELDS += fields 
```

Reply:
Not sure to see the benefit to write a method for that

Reply:
To factorize instead of rewriting the same 2 lines everywhere. Not a big deal if you don't do it btw.